static/src/javascripts/bootstraps/enhanced/media-player.js

```diff
@@ -347,6 +348,18 @@ const initPlayer = (withPreroll: boolean): void => {
     });
 };
 
+const sendToOphan = () => {
+    console.log('ophan event old player loaded');
+    ophan.record({
+        componentEvent: {
```

Comment:
I'm not sure audio events use the `componentEvent` model, yesterday @tomrf1 suggested otherwise. Worth comparing with the existing player.

Reply:
Yes this is leftover code that should be removed. Will do